internal/host/hostcommands/install_cmd_test.go

```diff
@@ -743,13 +746,17 @@ var _ = Describe("construct host install arguments", func() {
 				{
 					"name": "eth1",
 					"ipv4_addresses":["10.56.20.80/25"]
+				},
+				{
+					"name": "eth2",
+					"ipv4_addresses":["10.56.21.80/25"]
 				}
 			]
 		}`, models.DriveTypeISCSI, models.DriveTypeSSD)
```

Comment:
you check if `installationDisk.DriveType == models.DriveTypeMultipath` in line 319
but I don't see a test for it

Reply:
it's existing behavior, I didn't check if there are tests for it